### PR TITLE
i18n(es): update framework-components

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Update `framework-components.md`

### CODE REVIEW

File: src/pages/es/core-concepts/framework-components.md [hunk 1]
```diff
@@ -200,6 +200,22 @@ Esto te permite crear "aplicaciones" completas usando tu framework de JavaScript
 Los componentes de Astro siempre se renderizan a HTML estático, incluso cuando incluyen componentes de framework que son hidratados. Esto significa que solo se pueden pasar props que no hacen ninguna renderización a HTML. Pasar "render props" de React a componentes del framework desde un componente de Astro no funcionará, porque los componentes de Astro no pueden proporcionar el renderizado que este patrón requiere. En su lugar, utiliza slots con nombre.
 :::
 
+## ¿Puedo usar componentes de Astro dentro de mis componentes de framework?
+
+Cualquier componente de framework se convierte en una "isla" de ese framework. Estos componentes deben escribirse en su totalidad como código válido para ese framework, utilizando solo sus propios imports y paquetes. No puedes importar componentes `.astro` en un componente del framework (por ejemplo, `.jsx` o `.svelte`).
```
Comment (agustinmulet) [inline src/pages/es/core-concepts/framework-components.md:205]:
```suggestion
Cualquier componente de framework se convierte en una "isla" de ese framework. Estos componentes deben escribirse en su totalidad como código válido para ese framework, utilizando solo sus propios imports y paquetes. No puedes importar componentes `.astro` en un componente de framework (por ejemplo, `.jsx` o `.svelte`).
```

**Review by agustinmulet — COMMENTED** [review]
Small suggestion, aside from that, LGTM!

**Comment by kevinzunigacuellar** [comment]
Good to merge 🎉

**Comment by Yan-Thomas** [comment]
Thanks for this @kevinzunigacuellar, @agustinmulet! :purple_heart:

**Pull request merged**